uint16 START = 0
uint16 ATLOC1 = 1
uint16 ATLOC2 = 2
uint16 GRASPED_OBJ = 3
uint16 PLACED_OBJ = 4

uint16 state
